lib/grape/error_formatter/base.rb

```diff
@@ -4,7 +4,7 @@ module Grape
   module ErrorFormatter
     class Base
       class << self
-        def call(message, backtrace, options = {}, env = nil, original_exception = nil)
+        def call(message, backtrace, options = {}, env = nil, original_exception = nil, _status = nil) # rubocop:disable Metrics/ParameterLists
```

Comment:
Use `rubocop -a ; rubocop --auto-gen-config` instead of an inline `#rubocop:disable`, we have a lot of violations.

Reply:
done